query/src/sessions/session_settings.rs

```diff
@@ -74,66 +74,74 @@ impl Settings {
         let values = vec![
             // max_block_size
             SettingValue {
-                default_value:DataValue::UInt64(10000),
+                default_value: DataValue::UInt64(10000),
```

Comment:
:+1: 
Not sure why `make lint` not work for it :crying_cat_face:

Reply:
Clion can format this code. But I cannot find relevant configuration from https://rust-lang.github.io/rustfmt :(

Reply:
It seems to triggered some bugs : ( 

https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=col#space_after_colon